src/content/docs/2/guide/frontend/nuxt.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Nuxt
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Nuxt is a meta-framework for Vue.
+
+## Checklist
+
+- Use SSG by setting `ssr: false`. Tauri doesn't support server based solutions.
+- Set host to `0.0.0.0`.
+- Use `dist/` as `distDir` in `tauri.conf.json`.
+- Compile using `nuxi generate`.
+
+## Example Configuration
+
+<Tabs>
+	<TabItem label="npm">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "npm run dev",
+		"beforeBuildCommand": "npm run generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+	<TabItem label="yarn">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "yarn dev",
+		"beforeBuildCommand": "yarn generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+	<TabItem label="pnpm">
+	
+```json
+// tauri.conf.json
+{
+	"build": {
+		"beforeDevCommand": "pnpm dev",
+		"beforeBuildCommand": "pnpm generate",
+		"devPath": "http://localhost:3000",
+		"distDir": "../dist"
+	}
+}
+```
+	
+	</TabItem>
+</Tabs>
+
+```ts
+export default defineNuxtConfig({
+	// (optional) Enable the Nuxt devtools
+	devtools: { enabled: true },
+	// Enable SSG
+	ssr: false,
+	// (optional) Disable telemetry
+	telemetry: false,
+	vite: {
+		// Better support for Tauri CLI output
+		clearScreen: false,
+		// Enable environment variables
+		envPrefix: ['VITE_', 'TAURI_'],
```

Comment:
That part I took from the existing Vite docs in the next branch, haven't used this part myself but supposedly it helps.

Reply:
Not required but convenient to show, otherwise people won't know about it

Reply:
What's "convenient" mean in this case? 😅 By not having it would it cause issues for a dev when using Tauri? If that's the case then awesome to keep it in.

Reply:
Doesn't cause an issue but the info would be kinda lost and can't search for it since this is the only place that shows it?

Would be obsolete when I cleanup https://github.com/tauri-apps/tauri/blob/dev/tooling/cli/ENVIRONMENT_VARIABLES.md and make the naming consistent in v2, then we can have a page for these env vars similar to this https://doc.rust-lang.org/cargo/reference/environment-variables.html

Reply:
Ahhh I never knew that page existed! I think a page like that would be better to keep in the `tauri` repo so that it's more easily kept accurate and then we can link to it from here. Would that work? https://github.com/tauri-apps/tauri/blob/next/tooling/cli/ENVIRONMENT_VARIABLES.md

If that's okay then we can adapt the docs here to highlight that.

Reply:
We can't link to it, we need to index it and make it searchable, just need to polish it up a bit on tauri's side and then it can be copied to the docs on new releases.

Reply:
Ahhhhh... yeah good point, let me stub that page then and we can link to at least the stub in this PR.

Reply:
There, added that in now and marked it as a stub so we know it's a TODO for the docs.